.github/workflows/pysa.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           echo "CAML_LD_LIBRARY_PATH=$HOME/.opam/4.14.0/lib/stublibs:$HOME/.opam/4.14.0/lib/ocaml/stublibs:$HOME/.opam/4.14.0/lib/ocam" >> $GITHUB_ENV
           echo "$HOME/.opam/4.14.0/bin" >> $GITHUB_PATH
           echo "/home/opam/.opam/4.14.0/bin" >> $GITHUB_PATH
+          echo "OPAMCLI=2.1" >> $GITHUB_ENV
```

Comment:
Could you remove that line? It shouldn't be required

Reply:
I already tried before, but it failed.

Reply:
Perhaps the priority of environment variables is higher than the option `-- cli=2.1`?

Reply:
As I said above, this get the following warning:
```
Warning:  OPAMCLI should only ever be set to 2.0 - use '--cli=2.1' instead.
```
It sounds like `OPAMCLI=2.1` might not work correctly.